rpa-builder/6.6/modules/ROOT/pages/toolbox-file-operations-extract-pages-from-pdf-file.adoc

```diff
@@ -0,0 +1,63 @@
+= Extract Pages from PDF File
+
+Extract pages from a PDF file, customize the output, and create a new PDF file with the configured settings.
+
+== Properties
+
+* *File Settings*

```

Comment:
No description under this one?

Reply:
This is not a property, it's the UI container that groups the next properties.

Reply:
Should the nesting reflect that?

Reply:
Yes, you are right. Fixed it.